Assets/Scripts/UI/Menu/CreateRoom.cs

```diff
@@ -43,7 +42,11 @@ public void Start()
 
         public void Create()
         {
-            //TODO: This will fail when multiple input fields are present.
+            // TODO: Find a better place to put this.
```

Comment:
Find a better place for it, so why isn't a better place found yet?

Reply:
I'd like some help, because I'm not very familiar with the code.
Where is the code for loading a level?
Just in general, no matter whether you're creating a game or joining one.
I want to set the cursor mode for loading a level, so it has to run before the level starts loading.

Reply:
The level itself is loaded via OnJoinedRoom inside the GameManager, with PhotonNetwork.LoadLevel(Level.SceneName);

Reply:
Done.